sci-physics/hepstats/hepstats-0.8.1.ebuild

```diff
@@ -0,0 +1,28 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 pypi
+
+DESCRIPTION="Statistics tools and utilities."
+HOMEPAGE="
+	https://scikit-hep.org/hepstats/
+	https://github.com/scikit-hep/hepstats
+"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="test" # needs zfit
+
+RDEPEND="
+	dev-python/asdf[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/pandas[${PYTHON_USEDEP}]
+	dev-python/scipy[${PYTHON_USEDEP}]
+	dev-python/tqdm[${PYTHON_USEDEP}]
+	dev-python/uhi[${PYTHON_USEDEP}]
+"
+BDEPEND="
+	${RDEPEND}
+"
```

Comment:
And here too

Reply:
Thanks, I'll review the `*DEPEND`s.

Reply:
Since they are (mostly) 100% python packages (No C code or dependencies), I followed https://projects.gentoo.org/python/guide/basic.html#dependencies-in-python-packages and just put them as `RDEPEND`. I tested that they still build, by uninstalling the `RDEPEND`s and then run `ebuild ... install`.